### ISSUE TITLE
Is that supposed to work with Android version?

### ISSUE THREAD

**Issue by philfleck** [jekhor/yogabook-linux#29]
or are there any other workarounds to get the Adroid version of the Tablet to a useful state (newer Android or Linux or whatever?)

many thanks.

**Comment by jekhor** [comment]
No, unfortunately. I met instruction in the internet which describe how to replace ACPI tables by taken from Windows version of the tablet to get Windows running on the Android version, this may works for Linux also.

**Comment by philfleck** [comment]
thx for the reply. Such a pitty that they let such a nice device die. My Android version has almost become useless at this point, so i was looking for alternatives ROMs or such.

**Comment by ShinyCoconut** [comment]
> thx for the reply. Such a pitty that they let such a nice device die. My Android version has almost become useless at this point, so i was looking for alternatives ROMs or such.

They've managed to get Windows running on it (XDA post). I just do not know if the Halo Keyboard is working. You can try and let me know later hahah. Came across one of those selling for 300 BRL (60 USD), felt like I needed it haha.
